src/frontends/ir/src/utils.cpp

```diff
@@ -22,4 +22,60 @@ bool getStrAttribute(const pugi::xml_node& node, const std::string& name, std::s
     value = std::string(attr.value());
     return true;
 }
+
+Dimension strToDimension(const std::string& value) {
+    if (value.find('?') != std::string::npos) {
+        return {-1};
+    }
+    if (value.find("..") == std::string::npos) {
+        return {stringToType<int64_t>(value)};
+    }
+    std::string min_value_str = value.substr(0, value.find(".."));
+    int64_t min_value;
+    if (min_value_str.find_first_not_of(' ') == std::string::npos)
+        min_value = 0;
+    else
+        min_value = stringToType<int64_t>(min_value_str);
+
+    std::string max_value_str = value.substr(value.find("..") + 2);
+    int64_t max_value;
+    if (max_value_str.find_first_not_of(' ') == std::string::npos)
+        max_value = -1;
+    else
+        max_value = stringToType<int64_t>(max_value_str);
+
+    return {min_value, max_value};
+}
+
+PartialShape strToPartialShape(const std::string& value) {
```

Comment:
But what if we get shape like `...,...` ? Yes it is invalid but we will work with it. 
I think this function should guaranty that we are working only with valid shape form.

Also as this functions are major ones we need to cover them with tests (positive and negative).